index.html

```diff
@@ -68,23 +68,23 @@ <h2 class="card__question">What property flips the axes in flexbox?</h2>
           <li class="active">
             <a href="/index.html"
```

Comment:
to be consistent, you could add a "." here
```suggestion
            <a href="./index.html"
```